package.json

```diff
@@ -20,6 +22,7 @@
   },
   "homepage": "https://github.com/metamaps/metamaps#readme",
   "dependencies": {
+    "action-cable-node": "Connoropolous/action-cable-node",
```

Comment:
the main repo itself seems pretty sketchy... since there was a TYPO! in the code, but I was looking for a solution, and this was one I found. I do have a PR in to get it fixed, but haven't heard back

Reply:
damn

Reply:
if you do hear back, let's ask to be collaborators on github/npm so we can push fixes